test/e2e-test/regression-test/data/hl7v2/OML_O21.hbs/OML_O21.hbs-MDHHS-OML-O21-2.hl7.json

```diff
@@ -26,21 +26,35 @@
                 "address": [
                     {
                         "line": [
-                            "123 Main"
-                        ]
+                            "123 Main Street",
+                            "Apartment 3-C"
+                        ],
+                        "city": "Oldtown",
+                        "state": "MI",
+                        "postalCode": "48917",
+                        "country": "USA"
                     }
-                ]
+                ],
+                "telecom": [
```

Comment:
How are we getting no telecom detail, but only "use" code? Can we do a conditional check?